### PR TITLE
Casey seed attempt

### PR DESCRIPTION
**Pull request by cmacaulay**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 db/csv/*.csv
+app/views/amara_parkinglot
```
Comment (cmacaulay) [inline .gitignore:2]: @pixapi I saved your layout file here.

File: app/public/css/application.css [hunk 2]
```diff
@@ -1,150 +0,0 @@
-
-/*===== Custom styles =====*/
```
Comment (cmacaulay) [inline app/public/css/application.css:2]: Going to try materialize! Deleted CSS folders.

File: db/seeds.rb [hunk 3]
```diff
@@ -1,29 +1,45 @@
...
 
-  def transform_date(row)
-    date = row[:installation_date]
+  def transform_date(date)
     Time.strptime(date, "%m/%d/%Y").strftime("%d/%m/%Y")
   end
 
-  populate_stations
\ No newline at end of file
+  def import_trips_from_csv
```
Comment (cmacaulay) [inline db/seeds.rb:27]: This is the new seed method.

**Review by pixapi — APPROVED** [review]
stations.csv is present in github, we should find the way for it not to and don't push it to master @cmacaulay @maxglassie @codyborders

**Pull request merged**